scripts/components/OpenSearch-Dashboards/build.sh

```diff
@@ -95,6 +95,18 @@ case $PLATFORM-$DISTRIBUTION-$ARCHITECTURE in
         BUILD_PARAMS="build-platform"
         EXTRA_PARAMS="--skip-os-packages"
         SUFFIX="$PLATFORM-x64"
+    linux-deb-arm64)
+        TARGET="--$DISTRIBUTION"
+        EXT="$DISTRIBUTION"
+        BUILD_PARAMS="build"
+        EXTRA_PARAMS="--skip-archives"
+        SUFFIX="arm64"
+    linux-deb-x64)
+        TARGET="--$DISTRIBUTION"
+        EXT="$DISTRIBUTION"
+        BUILD_PARAMS="build"
+        EXTRA_PARAMS="--skip-archives"
+        SUFFIX="amd64"
```

Comment:
Use `--deb` for deb x64 and `--deb-arm` for deb arm64:
* https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1285

Reply:
`$DISTRIBUTION` is in this case `deb`, I added the suffix `-arm` to it https://github.com/opensearch-project/opensearch-build/pull/2526/commits/99a7db3679fa3907b5c9ab16fb09e24340a66f9a